geode-core/src/test/java/org/apache/geode/internal/process/AbstractProcessStreamReaderIntegrationTest.java

```diff
@@ -149,7 +147,7 @@ protected void givenStartedProcessWithStreamListeners(final Class<?> mainClass)
   }
 
   protected ConditionFactory await() {
-    return Awaitility.await().atMost(WAIT_FOR_READER_IS_RUNNING_TIMEOUT_MILLIS, MILLISECONDS);
+    return Awaitility.await().atMost(2, MINUTES);
```

Comment:
Did you mean to specify READER_JOIN_TIMEOUT_MILLIS instead of 2?  Otherwise it looks like that variable is only read by tests.